lib/tasks/redis.rake

```diff
@@ -83,41 +86,99 @@ namespace :redis do
     end
   end
 
-  desc 'Audit User Attributes'
-  task audit: :environment do
-    count = 0
-    mhv_users = 0
-    vha_patients = 0
-    mhv_non_patient = 0
-    patient_non_mhv = 0
-    addressees = 0
-    namespace = 'mvi-profile-response'
-    redis = Redis.current
-    redis.scan_each(match: "#{namespace}:*") do |key|
-      begin
-        resp = Oj.load(redis.get(key))[:response]
-        count += 1
-        mhvu = !resp.profile.mhv_ids.blank?
-        patient = patient?(resp.profile.vha_facility_ids)
-        mhv_users += 1 if mhvu
-        vha_patients += 1 if patient
-        mhv_non_patient += 1 if mhvu && !patient
-        patient_non_mhv += 1 if patient && !mhvu
-        addressees += 1 if addressee?(resp.profile.address)
-      rescue
-        puts "Couldn't parse #{key}"
+  namespace :audit do
+    desc 'Audit MHV/MVI User Attributes'
+    task mvi: :environment do
+      count = 0
+      mhv_users = 0
+      vha_patients = 0
+      mhv_non_patient = 0
+      patient_non_mhv = 0
+      addressees = 0
+
+      namespace = 'mvi-profile-response'
+      redis = Redis.current
+      redis.scan_each(match: "#{namespace}:*") do |key|
+        begin
+          resp = Oj.load(redis.get(key))[:response]
+          count += 1
+          mhvu = !resp.profile.mhv_ids.blank?
+          patient = patient?(resp.profile.vha_facility_ids)
+          mhv_users += 1 if mhvu
+          vha_patients += 1 if patient
+          mhv_non_patient += 1 if mhvu && !patient
+          patient_non_mhv += 1 if patient && !mhvu
+          addressees += 1 if addressee?(resp.profile.address)
```

Comment:
then here you could just do `counts_hash[:mhv_users] += 1 if mhvu` or possibly even do it all programmatically provided that the keys in your counts_hash could be variations on the key you're referencing from the `redis.scan_each`

NITPICK